test/test_base.py

```diff
@@ -145,13 +154,20 @@ def wait_for_all_tanks_status(self, target="running", timeout=20*60, interval=5)
         def check_status():
             tanks = self.wait_for_rpc("network_status", {"network": self.network_name})
             stats = {
-                "total": len(tanks)
```

Comment:
Why this change?

Is this so we account for lightning containers too? or something else

Reply:
Correct, it's because the total is number of containers which isn't 1 to 1 with number of tanks when we have lightning